doc/ReleaseNotes.md

```diff
@@ -18,6 +18,10 @@ r151028 release repository: https://pkg.omniosce.org/r151028/core
 * bhyve virtual machines are now fully supported and recommended in place of
   KVM - see <https://omniosce.org/info/bhyve> for more details.
 
+* bhyve and kvm branded zones are now available. Our website contains
+  [documentation](https://omniosce.org/info/bhyve_kvm_brand) on how to make
```

Comment:
that page does not exist (yet)

Reply:
I know, but it will before release (or we'll pick it up when we do the final checks). I have most of the content ready in the pkg5/doc area.